### DISCUSSION TITLE
still active?

### DISCUSSION THREAD

**Discussion by jahnj0584** [winder/Universal-G-Code-Sender#1946] [Polls]
Im still running Ver Apr 18 2018. Has anything major changed since? Do I offer suggestions?

**Comment by breiler** [comment]
There has been 811 commits since 2018-04-18, so it depends what you define as major and if it is of importance for you.

For instance, I personally think that the designer tool is pretty major: https://github.com/winder/Universal-G-Code-Sender/wiki/Usage#designer
The changes that Will has worked on to bring more axes (ABC) to the software is also quite significant. @bertieconfundo has helped with the continuous jogging so that it will no longer hang.
